php/.editorconfig

```diff
@@ -0,0 +1,15 @@
+root = true
```

Comment:
does php need to have this file? it seems an editor config, right? if so could you remove it, please?

Reply:
It ships with the lumen PHP framework:
- https://github.com/laravel/lumen/blob/8.x/.editorconfig

Reply:
I removed it because there is a `make code-check` command for lint anyway 👍  

(But usually PHP engineers just leave it as it comes with the framework skeleton project)